guides/common/modules/con_registering-a-centos-stream-host.adoc

```diff
@@ -1,4 +1,4 @@
 [id="Registering_a_CentOS_Stream_Host_{context}"]
-= Registering a CentOS Stream Host to {Project}
+= Registering a CentOS stream host to {Project}
```

Comment:
```suggestion
= Registering a CentOS Stream host to {Project}
```

Reply:
Fixed. This is a really weird module....